backend/app/repositories/user_repository.py

```diff
@@ -109,22 +110,21 @@ async def update_user(
     return _to_user_entity(user)
 
 
-async def get_user_list_from_organization(
-    user: User, session: AsyncSession
-) -> Union[list[User], User]:
-    """
+async def get_user_list_by_org(
+    user: UserEntity, session: AsyncSession
+) -> Union[List[UserEntity], UserEntity]:
+    """Add commentMore actions
     Return all users in the same organization, or the user itself if
     no organization.
     Superusers receive all users.
     """
     if user.is_superuser:
-        query = select(User)
+        stmt = select(User).options(selectinload(User.organization))
     elif user.organization_id is None:
         return user
```

Comment:
```suggestion
    elif user.organization_id is None:
        return [user]
```

you should always return the same type - this way, the frontend needs to manually check: is it only an instance or is it an array/list that is returned. avoid this by simply always returning a list, and if there is only one user: the frontend doesnt care, it iterates over the array/list of users and if there's only one, it shows only one.